libraries/AP_InertialSensor/AP_InertialSensor_tempcal.cpp

```diff
@@ -517,7 +517,7 @@ void AP_InertialSensor::get_persistent_params(ExpandingString &str) const
         }
 #if INS_AUX_INSTANCES
         for (uint8_t i=0; i<INS_AUX_INSTANCES; i++) {
-            const uint8_t imu = i+(INS_MAX_INSTANCES-INS_AUX_INSTANCES);
+            const uint8_t imu = i+(INS_MAX_INSTANCES-INS_AUX_INSTANCES)+1;
```

Comment:
We account for the imu offset for the base first 3 IMUs [just above](https://github.com/ArduPilot/ardupilot/pull/26624/files#diff-21e03d42136d362ede915596eaa38953235ca985dc629dbb91b27470f14562d9R502):